### PR TITLE
Squiggles Background Effect

### PR DESCRIPTION
**Pull request by ajpal**
Credit to Gaby Pulido for idea and initial implementation
squiggles

### CODE REVIEW

File: src/Effects.js [hunk 1]
```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
...
+      init: function () {
+        this.points = [];
+        p5.noStroke();
+        p5.angleMode(p5.DEGREES);
+        let numPoints = p5.height / this.dotSpacing;
+        for (var i = 0; i < numPoints; i++) {
+          this.points.push({
+            x: 0,
+            y: i * this.dotSpacing,
+            theta: 360 / this.period * i * this.dotSpacing,
```
Comment (epeach) [inline src/Effects.js]: What is the theta?

File: src/Effects.js [hunk 2]
```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
...
+      dotRadius: 7,
+      numSquiggles: 8,
+      init: function () {
+        this.points = [];
+        p5.noStroke();
+        p5.angleMode(p5.DEGREES);
+        let numPoints = p5.height / this.dotSpacing;
+        for (var i = 0; i < numPoints; i++) {
+          this.points.push({
+            x: 0,
```
Comment (epeach) [inline src/Effects.js]: Do all points start at the bottom of the page?

Comment (ajpal) [reply src/Effects.js]: `point.x` is set each tick of the draw loop, so setting to 0 here just to initialize it to something.

File: src/Effects.js (hunk 1 above)
Comment (ajpal) [reply src/Effects.js]: Theta is the current angle of the sine curve for each dot

**Pull request merged**